common/shm.h

```diff
@@ -71,7 +71,7 @@ extern "C"
     typedef char shm_name_t[SHM_NAME_MAX];
     typedef shm_name_t jack_shm_id_t;
 
-#elif WIN32
+#elif defined WIN32
```

Comment:
Does this compile

Reply:
I'm surprised it does, looking at this :P but it sure wont behave like intended, I assume